src/Generator/Generators/CSharp/CSharpSources.cs

```diff
@@ -364,6 +363,55 @@ public override bool VisitClassDecl(Class @class)
 
                 if (@class.IsDynamic)
                     GenerateVTable(@class);
+            } else
```

Comment:
All of this code looks identical to the one in the not opaque case so why duplicate it instead of just removing the check?